.github/workflows/CI.yml

```diff
@@ -0,0 +1,115 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - 'master'
+      - "maintenance/.+"
+  pull_request:
+    branches:
+      - 'master'
+      - "maintenance/.+"
+  schedule:
+    # Nightly tests run on master by default:
+    #   Scheduled workflows run on the latest commit on the default or base branch.
+    #   (from https://help.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events-schedule)
+    - cron: '0 0 * * *'
+
+jobs:
+  test:
+
+    runs-on: ${{ matrix.cfg.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - 3.6
+          - 3.7
+          # Uncomment when ready ;)
+          # - 3.8
+        cfg:
+          - os: ubuntu-latest
+            rdkit: true
+            openeye: false
+
+          - os: ubuntu-latest
+            rdkit: false
+            openeye: true
+
+          - os: ubuntu-latest
+            rdkit: true
+            openeye: true
+
+          - os: macOS-latest
+            rdkit: true
+            openeye: false
+
+    env:
+      ORGNAME: omnia
+      PACKAGENAME: openforcefield
+      OPENEYE_CHANNEL: -i https://pypi.anaconda.org/OpenEye/simple
+      PIP_ARGS: "-U"
+      CI_OS: ${{ matrix.cfg.os }}
+      RDKIT: ${{ matrix.cfg.rdkit }}
+      OPENEYE: ${{ matrix.cfg.openeye }}
+      PYVER: ${{ matrix.python-version }}
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Additional info about the build
+      shell: bash
+      run: |
+        uname -a
+        df -h
+        ulimit -a
+    - name: Fix conda permissions in MacOS
+      shell: bash
+      if: startsWith(matrix.cfg.os, 'macOS')
+      run: |
+        sudo chown -R $(id -u):$(id -g) ${CONDA}
+    - name: Configure conda
+      shell: bash
+      run: |
+        . devtools/gh-actions/initialize_conda.sh
+        conda activate
+        conda config --set always_yes yes
+        conda config --add channels omnia --add channels conda-forge --add channels openeye
+        conda install --quiet conda conda-build jinja2 anaconda-client
+        conda update --quiet --all
+        conda info --all
+        conda list
+    - name: Decrypt OpenEye license
+      shell: bash
+      env:
+        ENC_OE_LICENSE: ${{ secrets.OE_LICENSE__EXP_JUN2020 }}
+      run: |
+        echo "${ENC_OE_LICENSE}" > $HOME/oe_license.txt
+    - name: Prepare test environment
+      shell: bash
+      run: |
+        . devtools/gh-actions/initialize_conda.sh
+        conda activate
+        python devtools/scripts/create_conda_env.py -n=test -p=$PYTHON_VER devtools/conda-envs/test_env.yaml
```

Comment:
I think this should be `$PYVER `?

Reply:
Ohh, excellent catch! So sorry, this could have been such a terrible mistake in `master`. Thanks a bunch!